Sources/FioriSwiftUICore/_FioriStyles/BannerMultiMessageSheetStyle.fiori.swift

```diff
@@ -422,3 +422,61 @@ extension BannerMultiMessageSheetFioriStyle {
         }
     }
 }
+
+struct BannerMultiMessageSheetModifier: ViewModifier {
+    @Binding var isPresented: Bool
+    let title: () -> any View
+    let closeAction: () -> any View
+    /// callback when this component want to dismiss itself
+    let dismissAction: (() -> Void)?
+    /// callback when category or single item is removed
+    let removeAction: ((String, UUID?) -> Void)?
+    /// callback when the link button is clicked
+    let viewDetailAction: ((UUID) -> Void)?
+    /// the mark to turn on section header or not
+    let turnOnSectionHeader: Bool
+    /// view for each item under the category
+    let messageItemView: (UUID) -> any View
+    /// the data source for banner multi-message sheet
+    @Binding var bannerMultiMessages: [BannerMessageListModel]
+    
+    func body(content: Content) -> some View {
+        content
+            .popover(isPresented: self.$isPresented, content: {
+                BannerMultiMessageSheet(title: self.title, closeAction: self.closeAction, dismissAction: {
+                    self.isPresented = false
+                    self.dismissAction?()
+                }, removeAction: self.removeAction, viewDetailAction: self.viewDetailAction, turnOnSectionHeader: self.turnOnSectionHeader, messageItemView: self.messageItemView, bannerMultiMessages: self.$bannerMultiMessages)
+                    .presentationDetents([.medium, .large])
+            })
+    }
+}
+
+//
+
+public extension View {
+    /// To show the BannerMultiMessageSheet in gentle way.
```

Comment:
```suggestion
    /// Control the BannerMultiMessageSheet to be shown or not
```
Feel free to reword the above. I am just not sure what "in gentle way" means here.